Frontend/ui-driver/src/Screens/PaymentHistoryScreen/Transformer.purs

```diff
@@ -91,6 +92,13 @@ buildTransactionDetails (API.HistoryEntryDetailsEntityV2Resp resp) gradientConfi
                     val : getFixedTwoDecimals $ case driverFee'.isCoinCleared of
                                                 true -> driverFee'.driverFeeAmount
                                                 false -> fromMaybe 0.0 driverFee'.coinDiscountAmount
+                  },
+                  {
+                    key : "GST",
+                    title : getString $ GST_WITH_PERCENTAGE $ maybe gstPercentage (\percentage -> show $ 100.0 * percentage) driverFee'.gstPercentage,
+                    val : case driverFee'.gst of
+                            Just gst -> if gst > 0.0 then getFixedTwoDecimals gst else ""
```

Comment:
if it is `""` then we shouldn't display this field itself right. Why to show empty string.